src/structures/MessageMentions.js

```diff
@@ -100,7 +100,10 @@ class MessageMentions {
    */
   get channels() {
     if (this._channels) return this._channels;
-    if (!this._guild) return null;
+    if (!this._guild) {
+      this._channels = new Collection();
```

Comment:
Wouldn't it be better to just put `this._channels = new Collection();` above the `if (!this._guild)`, since the code after it would do the same?